core/src/commonMain/kotlin/com/ustadmobile/core/torrent/TorrentDownloadListener.kt

```diff
@@ -0,0 +1,8 @@
+package com.ustadmobile.core.torrent
+
+interface TorrentDownloadListener {
+
+    fun onComplete()
+
+    fun onProgress(progress: Int)
```

Comment:
Please add a comment here for the progress value. What does this represent? Is this from 0-100, or what is it?